### PR TITLE
Fix: install django/twisted versions override

### PR DESCRIPTION
**Pull request by v-andrius**
## Description
django/twisted versions override has been ignored, because of `only_if` block:
https://github.com/sous-chefs/graphite/blob/v1.1.1/recipes/_web_packages.rb#L25-L31
https://github.com/sous-chefs/graphite/blob/v1.1.1/recipes/_carbon_packages.rb#L29-L36

### Issues Resolved
Django/Twisted version override ignored [#318](https://github.com/sous-chefs/graphite/issues/318)

### Known issues
Even if right `django` version installed, package is upgraded during `uwsgi` package installation:
https://github.com/sous-chefs/graphite/blob/v1.1.1/recipes/_web_packages.rb#L34

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## 1.1.2 - (2020-05-22)
+
+### Fixed
+
+- install Django/Twisted overrided version ([#318])
```
Comment (damacus) [inline CHANGELOG.md:9]:
```suggestion
- Fix Django/Twisted overridden version ([#318])
```

**Comment by damacus** [comment]
@Xorima is it time to remove the recipes from this cookbook yet?

**Comment by Xorima** [comment]
It needs to be done but it's on the list, I wish I had more time, for now i would just merge this in and ship it @damacus

**Pull request merged**